README.md

```diff
@@ -28,13 +28,17 @@ Features include:
 
 ### Current Status
 
-This library has matured quite a bit, but should still be considered incomplete. 
+> [!TIP]
+> MSRV Note: This crate is now on Rust `1.70.0`.
```

Comment:
This wording might be somewhat misleading, as there is currently no CI validating for `1.70`.

Reply:
Yes, I'd consider just removing the first sentence. Especially since it was always possible to use this crate with the newest version of Rust.